### PR TITLE
Update french portal.json

### PR DESCRIPTION
**Pull request by jeremielondon**
Added some french

Got some code for us? Awesome 🎊!

Please include a description of your change & check your PR against this list, thanks!

- [ ] There's a clear use-case for this code change, explained below
- [ ] Commit message has a short title & references relevant issues
- [ ] The build will pass (run `yarn test:all` and `yarn lint`)

We appreciate your contribution!

---

### <samp>🤖 Generated by Copilot at beef9e7</samp>

This pull request improves the French localization of the portal UI by adding or updating translations in the `fr/portal.json` file. This makes the portal more user-friendly and accessible for French-speaking users.

### CODE REVIEW

File: ghost/i18n/locales/fr/portal.json [hunk 1]
```diff
@@ -82,7 +82,7 @@
     "Not receiving emails?": "Vous n’avez pas reçu d’e-mails ?",
     "Now check your email!": "Veuillez maintenant vérifier votre boite de réception.",
     "Once resubscribed, if you still don't see emails in your inbox, check your spam folder. Some inbox providers keep a record of previous spam complaints and will continue to flag emails. If this happens, mark the latest newsletter as 'Not spam' to move it back to your primary inbox.": "",
-    "Permanent failure (bounce)": "",
+    "Permanent failure (bounce)": "Échec permanent (rebond)",
```
Comment (Mysterious-Dev) [inline ghost/i18n/locales/fr/portal.json:85]: Not sure about this translation :/

Comment (jeremielondon) [reply ghost/i18n/locales/fr/portal.json:85]: Yep fair enough

**Comment by Mysterious-Dev** [comment]
With the merge of #16942, this PR can be closed ?

**Comment by daniellockyer** [comment]
@jeremielondon Would you be able to rebase your PR/close if the PR is no longer needed? 🙂

**Comment by daniellockyer** [comment]
Closing this PR for now but ping me if you'd still like to make these changes 🙂

**Pull request closed without merging**